### PR TITLE
fix(config): adding some missing instances block device config

### PR DESCRIPTION
**Pull request by marchello2000**

### CODE REVIEW

**Comment by marchello2000** [comment]
@abelzberg please let me know if you have feedback

**Comment by marchello2000** [comment]
Added offline feedback from @abelzberg

File: clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/BlockDeviceConfig.groovy [hunk 1]
```diff
@@ -99,6 +99,11 @@ class BlockDeviceConfig {
       "i3.16xlarge" : enumeratedBlockDevicesWithVirtualName(8),
       "i3.metal"    : enumeratedBlockDevicesWithVirtualName(8),
 
+      "i3en.large"  : enumeratedBlockDevicesWithVirtualName(1),
+      "i3en.xlarge" : enumeratedBlockDevicesWithVirtualName(1),
+      "i3en.2xlarge": enumeratedBlockDevicesWithVirtualName(1),
```
Comment (cfieber) [inline clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/BlockDeviceConfig.groovy]:
```suggestion
      "i3en.2xlarge": enumeratedBlockDevicesWithVirtualName(2),
```

Comment (cfieber) [reply clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/BlockDeviceConfig.groovy]: I think, based on this table: https://aws.amazon.com/ec2/instance-types/i3en/

File: clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/BlockDeviceConfig.groovy [hunk 2]
```diff
@@ -124,14 +129,18 @@ class BlockDeviceConfig {
...
+      "m5.16xlarge" : sizedBlockDevicesForEbs(120),
       "m5.24xlarge" : sizedBlockDevicesForEbs(120),
 
       "m5d.large"    : enumeratedBlockDevicesWithVirtualName(1),
       "m5d.xlarge"   : enumeratedBlockDevicesWithVirtualName(1),
       "m5d.2xlarge"  : enumeratedBlockDevicesWithVirtualName(1),
       "m5d.4xlarge"  : enumeratedBlockDevicesWithVirtualName(1),
+      "m5d.8xlarge"  : enumeratedBlockDevicesWithVirtualName(2),
       "m5d.12xlarge" : enumeratedBlockDevicesWithVirtualName(2),
+      "m5d.16xlarge" : enumeratedBlockDevicesWithVirtualName(2),
```
Comment (cfieber) [inline clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/BlockDeviceConfig.groovy]:
```suggestion
      "m5d.16xlarge" : enumeratedBlockDevicesWithVirtualName(4),
```

https://aws.amazon.com/ec2/instance-types/m5/

File: clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/BlockDeviceConfig.groovy [hunk 3]
```diff
@@ -151,14 +160,18 @@ class BlockDeviceConfig {
...
+      "r5.16xlarge"  : sizedBlockDevicesForEbs(120),
       "r5.24xlarge" : sizedBlockDevicesForEbs(120),
 
       "r5d.large"    : enumeratedBlockDevicesWithVirtualName(1),
       "r5d.xlarge"   : enumeratedBlockDevicesWithVirtualName(1),
       "r5d.2xlarge"  : enumeratedBlockDevicesWithVirtualName(1),
       "r5d.4xlarge"  : enumeratedBlockDevicesWithVirtualName(2),
+      "r5d.8xlarge"  : enumeratedBlockDevicesWithVirtualName(2),
       "r5d.12xlarge" : enumeratedBlockDevicesWithVirtualName(2),
+      "r5d.16xlarge" : enumeratedBlockDevicesWithVirtualName(2),
```
Comment (cfieber) [inline clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/BlockDeviceConfig.groovy]:
```suggestion
      "r5d.16xlarge" : enumeratedBlockDevicesWithVirtualName(4),
```

https://aws.amazon.com/ec2/instance-types/r5/

File: clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/BlockDeviceConfig.groovy [hunk 4]
```diff
@@ -124,14 +129,18 @@ class BlockDeviceConfig {
...
       "m5.4xlarge"  : sizedBlockDevicesForEbs(120),
+      "m5.8xlarge"  : sizedBlockDevicesForEbs(120),
       "m5.12xlarge" : sizedBlockDevicesForEbs(120),
+      "m5.16xlarge" : sizedBlockDevicesForEbs(120),
       "m5.24xlarge" : sizedBlockDevicesForEbs(120),
 
       "m5d.large"    : enumeratedBlockDevicesWithVirtualName(1),
       "m5d.xlarge"   : enumeratedBlockDevicesWithVirtualName(1),
       "m5d.2xlarge"  : enumeratedBlockDevicesWithVirtualName(1),
       "m5d.4xlarge"  : enumeratedBlockDevicesWithVirtualName(1),
```
Comment (cfieber) [inline clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/BlockDeviceConfig.groovy]:
```suggestion
      "m5d.4xlarge"  : enumeratedBlockDevicesWithVirtualName(2),
```

File: clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/BlockDeviceConfig.groovy (hunk 1 above)
Comment (cfieber) [reply clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/BlockDeviceConfig.groovy]: worth adding the rest of the i3en types from that page while you are in here?

Comment (marchello2000) [reply clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/BlockDeviceConfig.groovy]: 👍

**Pull request merged**